app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+from decimal import Decimal
+import json
```

Comment:
I thik this is better import order:
```suggestion
import json

from decimal import Decimal
```